lib/oauth2-client.js

```diff
@@ -0,0 +1,113 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as storage from './storage';
```

Comment:
Always use file extensions when using ESM style imports to import files (as opposed to packages).

```suggestion
import * as storage from './storage.js';
```

Reply:
Updated.